wrapanapi/lenovo.py

```diff
@@ -307,3 +310,17 @@ def set_server_led_blinking(self, server_name):
         response = self.change_led_status(server, led['name'], 'Blinking')
 
         return "LED state action has been sent, status:" + str(response.status_code)
+
+    def stats(self, *requested_stats, **kwargs):
+        requester = None
```

Comment:
This an the lines below can all be simplified down to a single `dict.get()` call.  The default behavior for .get() is to return None as a default if the key isn't found.

``` python
kwargs.get('requester')